plm/plm.go

```diff
@@ -246,8 +246,7 @@ func (ml *PLM) Status(ctx context.Context) *Status {
 	}
 
 	if s.Repl.IsStarted() {
-		intialSyncLag := max(int64(s.Clone.FinishTS.T)-int64(s.Repl.LastReplicatedOpTime.T), 0)
-		s.InitialSyncLagTime = &intialSyncLag
+		s.InitialSyncLagTime = 0
```

Comment:
Do we need this line since we always get a fresh `status` and `InitialSyncLagTime` will have its zero value, which is `0`?

Reply:
fixed: 616335afe4193dff68d8436602e8c1acdfc46777